src/app/shared/confirm/confirm.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="container--confirm">
+  <p>
+    Are you sure you want to delete?
+  </p>
+  <button class="btn" (click)="sendConfirmation(true)">YES</button>
+  <button class="btn" (click)="sendConfirmation(false)">NO</button>
```

Comment:
Use mat-action to take yes or no input. Eg:
```html
<div mat-dialog-actions>
  <button mat-button (click)="onNoClick()">No Thanks</button>
  <button mat-button [mat-dialog-close]="data.animal" cdkFocusInitial>Ok</button>
</div>
```

Reply:
Made the changes in the code.